roxie/ccd/ccdcontext.cpp

```diff
@@ -1054,7 +1054,7 @@ class StreamedRawDataReader : public RawDataReader
 
             // Note: I am the only thread reading (we only support a single input dataset in roxiepipe mode)
             MemoryBuffer reply;
-            client.readBlock(reply, readTimeout);
+            client.readBlocktms(reply, readTimeout, maxBlockSize);
```

Comment:
This is now a time out of 300ms instead of 5 minutes.  Is that correct?   I wonder if it is meant to be long for this call.

Reply:
It was actually infinite before, I think, though that was not intentional.

300ms may be a bit short though.

Reply:
Actually the parameter order bug confused me I think.

Fixed.